pages/desktop/developers/addons_manage.py

```diff
@@ -15,3 +28,23 @@ def my_addons_page_logo(self):
     @property
     def my_addons_page_title(self):
         return self.find_element(*self._page_title_locator)
+
+    def sort_by_created(self):
+        self.find_element(*self._sort_by_created_locator).click()
```

Comment:
Yes, there will be a few more sorting options. Initially I considered identifying separate elements for each of them. I could also identify them as a list of items and then access them by their index, since they are in the same component.